### ISSUE TITLE
monitor number issue

### ISSUE THREAD

**Issue by Gagacs007** [rustdesk/doc.rustdesk.com#471]
The latest update has swapped the monitors. If I set monitor 2 as a single display on the remote machine, rustdesk won't connect. It's trying to connect to a non-existent monitor. Multiple display mode is also turned off.
![Képernyőkép 2024-12-10 085308](https://github.com/user-attachments/assets/fdc28305-82ce-43ff-abec-bff69a307e98)

![Képernyőkép 2024-12-10 084558](https://github.com/user-attachments/assets/a5064959-c1a9-4c02-a5a5-17d32778536d)

**Comment by rustdesk** [comment]
@fufesou

**Comment by rustdesk** [comment]
> If I set monitor 2 as a single display on the remote machine

What does this mean?

**Comment by Gagacs007** [comment]
> > If I set monitor 2 as a single display on the remote machine
> 
> What does this mean?

I currently have two monitors on the remote machine. If I set one monitor on it, rustdesk cannot display an image.

**Comment by Gagacs007** [comment]
I kept changing the monitor settings on the remote machine until it reset.
![Képernyőkép 2024-12-10 090912](https://github.com/user-attachments/assets/6b7551a8-089b-4d89-a0f7-ca90c2847772)

**Comment by Gagacs007** [comment]
> @Gagacs007 Thanks for your feedback.
> 
> > The latest update has swapped the monitors.
> 
> The display's numbers are not the ones in the display settings? What version did you use before? Does it work fine?
> 
> > I currently have two monitors on the remote machine. If I set one monitor on it, rustdesk cannot display an image.
> > ...
> > I kept changing the monitor settings on the remote machine until it reset.
> 
> By disabling it in the display settings?
> 
> > Multiple display mode is also turned off.
> 
> What does it mean?

Yes. What is monitor 1 on the remote machine is shown as monitor 2 by rustdesk. Version 1.3.5.

What solved it was that on the remote machine I first set monitor 1 as a single display, then monitor 2. Then I set monitor 1 back to "Extend these displays".

Rustdesk display settings >>>
![Képernyőkép 2024-12-10 094935](https://github.com/user-attachments/assets/37b8b84d-65de-4ba7-9994-77f28c5f4041)

**Comment by rustdesk** [comment]
> If I set one monitor on it, rustdesk cannot display an image.

How to set?
